### PR TITLE
Blog App

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
- Added RSpec as a gem to my project.
- Create a Request spec file for all the controllers.
- Make sure that for each action you check:
- If response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject do
+    User.new(created_at: Time.now, updated_at: Time.now, name: 'User 1', photo: 'Random URL', bio: 'Random text',
+             posts_counter: 0)
+  end
+
```
Comment (emmanuellekamwa) [inline spec/models/user_spec.rb:8]: - [ ] The spec folder has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory with `type:request`. Check this example: 
 ```ruby
require 'rails_helper'

 RSpec.describe "Users", type: :request do
   ...
 end

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔🛑**
Hello @Isabellanamutenda 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Isabellanamutenda) [reply spec/models/user_spec.rb:8]: Made the changes

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @Isabellanamutenda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**